src/styles.css

```diff
@@ -527,9 +527,9 @@ cg-container cg-board::before {
 	cursor: grab !important;
 }
 
-piece {
+/*piece {
 	pointer-events: auto !important;
-}
+}*/
```

Comment:
I didn't quite understand what problem was fixed by commenting this out. At what point were square highlights broken before? On moves, premoves, the last move or something else?

Reply:
When making a move by dragging a piece, the destination square does not get highlighted. Commenting this out lets the destination square get highlighted when dragging a piece. From my testing, nothing else is effected since the point-events are set in other places.

Reply:
Ah ok, I hadn't seen that. Very good, thanks for the change and the explanation